datadog/resource_datadog_dashboard.go

```diff
@@ -4610,6 +4610,21 @@ func getTimeseriesDefinitionSchema() map[string]*schema.Schema {
 			Optional:     true,
 			ValidateFunc: validateTimeseriesWidgetLegendSize,
 		},
+		"legend_layout": {
+			Description:  "The layout of the legend displayed in the widget.",
```

Comment:
nit: We could add `One of "auto", "horizontal" or "vertical"` in the description here